CHANGELOG.md

```diff
@@ -1,287 +1,89 @@
-
-```ts
-const geometries: GeometryUnion[] = [polygon1, extent1, point1];
-for (const geometry of geometries) {
-  switch (geometry.type) {
-    case "polygon":
-      // Typescript correctly infers the geometry type is a polygon
-      const firstRing = geometry.rings.at(0);  
-      // work with first ring
-      break;
-    default:
-      // Handle the rest
-  }
-}
-```
-
-The SDK now consistently uses the union types instead of the base classes. All the new union types are suffixed with "Union", such as `GeometryUnion`, to clearly differentiate them from the base class names.  
-
-### Improved type safety
-
-By default, the typings now include `null` and `undefined` where appropriate. This enhances type safety and makes them compatible with projects where [`strict`](https://www.typescriptlang.org/tsconfig/#strict) or [`strictNullChecks`](https://www.typescriptlang.org/tsconfig/#strictNullChecks) are enabled. Without strict null checks, `null` or `undefined` can be assigned to any TypeScript type, potentially leading to runtime errors if your code tries to access a `null` or `undefined` variable.
-
-### Autocasting enhancements
-
-Support for using TypeScript property setters was added to autocasting. Types now work in constructors, parameters and setters. Autocasting allows JavaScript objects to be automatically cast to SDK class types without the need for explicit imports. In the following example, the types are correctly [narrowed](https://www.typescriptlang.org/docs/handbook/2/narrowing.html) when the renderer is set after the FeatureLayer is created:
-
-```ts
-let citiesLayer = new FeatureLayer({
-  url: "http://url.to.service",
-});
-
-// At 4.32 this works in TypeScript
-citiesLayer.renderer = {
-  type: "simple",  
-  symbol: {
-    type: "simple-marker", 
-    size: 6,
-    color: "black",
-    outline: { 
-      width: 0.5,
-      color: "white"
-    }
-  }
-};
-```
-
-### TypeScript migration recommendations
-
-When introducing these TypeScript changes into your applications there are a number of recommendations.
-
-When working with strict null:
-- Temporarily disable strict null to unblock the TypeScript upgrade, then gradually work on re-enabling it.
-- Use of `!` non-null assertion operator to quickly make your build pass and then gradually work on removing them.
-- Use of `?` operator to mark properties as optional.
-
-When working with unions:
-- Replace uses of base class types with `esri/unionTypes`.
-- Use [type guards](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#typeof-type-guards) to narrow the types when needed.
-
-When working with autocasting:
-- In many cases, it can be beneficial to use the actual class constructors. Then there is no need to specify `type`, or use `const` assertions, and the classes are auto-imported by Visual Studio Code and other code editors.
-- For more information refer to the Autocasting in TypeScript guide topic.
+## Component and widget updates
```

Comment:
If we have a section heading we should have at least some content for it.

Reply:
Removed empty headers